### ISSUE TITLE
KeyError: 'customFieldItems'

### ISSUE THREAD

**Issue by alexxa** [sarumont/py-trello#247]
```
Traceback (most recent call last):
  File "scripts/trello_requirements.py", line 242, in <module>
    main()
  File "scripts/trello_requirements.py", line 238, in main
    args.dry_run, args.lookup_method, args.approver)
  File "scripts/trello_requirements.py", line 144, in get_trello_board_content
    tcards = board.get_cards()
  File "/var/lib/jenkins/workspace/trello-to-polarion-igulina/Polarion_Scripts/.venv/lib/python2.7/site-packages/trello/board.py", line 302, in get_cards
    return list([Card.from_json(self, json) for json in json_obj])
  File "/var/lib/jenkins/workspace/trello-to-polarion-igulina/Polarion_Scripts/.venv/lib/python2.7/site-packages/trello/card.py", line 157, in from_json
    card.customFields = CustomField.from_json_list(card, json_obj['customFieldItems'])
KeyError: 'customFieldItems'
```

**Comment by alexxa** [comment]
@jamesf-brainlabs Hi, the traceback is still there, I have just checked it again.

**Comment by jamesf-brainlabs** [comment]
@alexxa 
I've looked back, and it seemed that the only change I made that was relevant to this, was that I was using `open_cards()`, instead of `get_cards()`, and I added that filter to `open_cards()`:

``` python
	def open_cards(self):
		"""Returns all open cards on this board

		:rtype: list of Card
		"""
		filters = {
			'filter': 'open',
			'fields': 'all',
			'customFieldItems': 'true'
		}
		return self.get_cards(filters)
```

**Comment by shatterstone** [comment]
I've gotten the same traceback, but mine occurs when trying to add a card... 
`Traceback (most recent call last):
  File "trello_test.py", line 13, in <module>
    my_list.add_card(name='test name', desc='desc card',position='top')
  File "/Library/Python/2.7/site-packages/trello/trellolist.py", line 105, in add_card
    return Card.from_json(self, json_obj)
  File "/Library/Python/2.7/site-packages/trello/card.py", line 157, in from_json
    card.customFields = CustomField.from_json_list(card, json_obj['customFieldItems'])
KeyError: 'customFieldItems'
`

**Comment by alexxa** [comment]
@sarumont @rbrady @shatterstone @jamesf-brainlabs  PR #248 doesn't fix it, unfortunately: 

```
+ pip install py-trello
Collecting py-trello
  Downloading https://files.pythonhosted.org/packages/4d/3e/21c9e3fd77b62ac67c67c2a540b6d2970c81bc39f09451ee42ad181aea83/py-trello-0.11.2.tar.gz
<...>
Traceback (most recent call last):
  File "scripts/trello_requirements.py", line 242, in <module>
    main()
  File "scripts/trello_requirements.py", line 238, in main
    args.dry_run, args.lookup_method, args.approver)
  File "scripts/trello_requirements.py", line 144, in get_trello_board_content
    tcards = board.get_cards()
  File "/var/lib/jenkins/workspace/trello-to-polarion-igulina/Polarion_Scripts/.venv/lib/python2.7/site-packages/trello/board.py", line 305, in get_cards
    return list([Card.from_json(self, json) for json in json_obj])
  File "/var/lib/jenkins/workspace/trello-to-polarion-igulina/Polarion_Scripts/.venv/lib/python2.7/site-packages/trello/card.py", line 157, in from_json
    card.customFields = CustomField.from_json_list(card, json_obj['customFieldItems'])
KeyError: 'customFieldItems'

```

**Comment by rbrady** [comment]
If you look at the traceback it's failing in a different method.  I filed a new issue on it and started a fix last night.  I'll create a new PR today after I test the fix.

**Comment by alexxa** [comment]
@rbrady in this issue report the traceback was in get_cards method and in my last comment it's still in get_cards method. And also I wouldn't open a new issue for every method =) but if you feel that way...

**Comment by rbrady** [comment]
The problem is still occurring because of the following line in the traceback:

card.customFields = CustomField.from_json_list(card, json_obj['customFieldItems'])

The code expects the ''customFieldItems" key will always be present in the underlying json returned.  This does not work in the case where the custom fields power up is not enabled on a board or the case when you attempt to retrieve a card that does not have any custom field values set.  I hit this when adding a new card.

Fix being tested.  PR imminent.

**Comment by rbrady** [comment]
See PR#250

**Comment by sarumont** [comment]
Merged #250 and pushed a `0.11.3` up to PyPi. Thanks!

**Comment by superghm** [comment]
I found the same issue in open_cards(), get_cards() and all_cards().
How can I solve this?

Traceback (most recent call last):
  File "teste.py", line 17, in <module>
    print(quadro.all_cards())
  File "C:\Users\Guilherme Mendes\AppData\Local\conda\conda\envs\virtualconda\lib\site-packages\trello\board.py", line 263, in all_cards
    return self.get_cards(filters)
  File "C:\Users\Guilherme Mendes\AppData\Local\conda\conda\envs\virtualconda\lib\site-packages\trello\board.py", line 302, in get_cards
    return list([Card.from_json(self, json) for json in json_obj])
  File "C:\Users\Guilherme Mendes\AppData\Local\conda\conda\envs\virtualconda\lib\site-packages\trello\board.py", line 302, in <listcomp>
    return list([Card.from_json(self, json) for json in json_obj])
  File "C:\Users\Guilherme Mendes\AppData\Local\conda\conda\envs\virtualconda\lib\site-packages\trello\card.py", line 157, in from_json
    card.customFields = CustomField.from_json_list(card, json_obj['customFieldItems'])
KeyError: 'customFieldItems'
